cli/flox/src/commands/init/node.rs

```diff
@@ -468,141 +467,86 @@ impl Node {
                 result.version.as_ref().map(|version|format!("version {version}")).unwrap_or("another version".to_string()));
                 (message, result.version.clone())
             },
-            (PackageJSONVersion::Unspecified, None) => {
+            (Some(PackageJSONVersion::Unspecified), None) => {
                 let result = Self::get_default_package("nodejs", flox)?;
                 mentions_package_json = true;
                 ("Flox detected a package.json".to_string(), result.version)
             },
             // get_action() returns NodeAction::Nothing for this case so it's unreachable
-            (PackageJSONVersion::None, None) => unreachable!(),
+            (None, None) => unreachable!(),
         };
         Ok((message, version, mentions_package_json))
     }
 
+    fn prompt(&self) -> Result<bool> {
```

Comment:
since we also have _another_ prompt, can you name this something more specific?
e.g.:

```suggestion
    fn single_option_prompt(&self) -> Result<bool> {
```

Reply:
Fixing